gapis/api/vulkan/replay.go

```diff
@@ -914,15 +918,13 @@ func replayProfile(ctx context.Context,
 	transforms = append(transforms, getCommonInitializationTransforms("ProfileReplay", true)...)
 
 	profileTransform := newEndOfReplay()
-	for _, rr := range rrs {
-		profileTransform.AddResult(rr.Result)
-		request := rr.Request.(profileRequest)
-		transforms = append(transforms, newWaitForPerfetto(request.traceOptions, request.handler, request.buffer, api.CmdID(len(initialCmds))))
-		transforms = append(transforms, newProfilingLayers(layerName))
-		transforms = append(transforms, newMappingExporter(ctx, request.handleMappings))
-	}
-
+	profileTransform.AddResult(rrs[0].Result)
+	request := rrs[0].Request.(profileRequest)
```

Comment:
All of these accesses to rrs[0] need to be inside an if len() > 0 right?

Reply:
The check is in the beginning of the function as an early return